content/docs/1.6.0/deploy/important-notes/index.md

```diff
@@ -106,3 +106,26 @@ Starting from `v1.5.0`, the following deprecated custom resource fields will be
 - BackingImageManager.status.backingImageFileMap.directory
 - BackingImageManager.status.backingImageFileMap.downloadProgress
 - BackingImageManager.status.backingImageFileMap.url
+
+### Engine Upgrade Enforcement
+
+Starting from `v1.6.0`, Longhorn will implement mandatory version upgrades for the engines. The minimum required supporting version will be outlined in the [release note](https://github.com/longhorn/longhorn/releases/tag/v{{< current-version >}}).
+
+When performing a helm upgrade, incompatible versions are automatically detected and prevented through a pre-hook mechanism. However, if you've installed Longhorn using manifests, the enforcement is handled by the longhorn-manager. This will result in unsuccessful pod launches and corresponding error logs. If you encounter such an error, it's necessary to revert to the previous version and upgrade any engine currently utilizing the incompatible engine image.
```

Comment:
```suggestion
When upgrading through Helm, a component compatibility check is automatically performed. If the new Longhorn Manager is not compatible with the engine images that are currently in use, the upgrade path is blocked through a pre-hook mechanism.

If you installed Longhorn using a manifest file, engine upgrades are enforced by Longhorn Manager. Attempts to upgrade Longhorn Manager may cause unsuccessful pod launches and generate corresponding error logs. If you encounter such errors, you must revert to the previous Longhorn Manager version and then upgrade engines using incompatible engine images.
```

Reply:
I like the suggestions, but there are some parts that seem a little confusing. For instance:
- `If the new Longhorn Manager is not compatible with the engine images that are currently in use`. We've implemented the check in the Longhorn manager to ensure compatibility across the entire Longhorn system.
- `you must revert to the previous Longhorn Manager version and then upgrade engines using incompatible engine images.` This is a bit confusing because we don't actually upgrade the engines using the engine images.

So based on your suggestions, I've modified it a little. PTAL.

Reply:
Thanks for clarifying. Your changes look good.